snap/README.md

```diff
@@ -39,7 +45,7 @@ $ sudo snap install --classic --dangerous kata-containers_[VERSION]_[ARCH].snap
 Replace `VERSION` with the current version of Kata Containers and `ARCH` with
 the system architecture.
 
-## Configuring Kata Containers ##
+## Configuring Kata Containers
```

Comment:
You might want to change this title to remove the gerund as we have in other docs: 
Configure Kata Containers